modules/foreman_setting.py

```diff
@@ -158,7 +142,7 @@ def main():
 
     entity_dict = sanitize_entity_dict(entity_dict, name_map)
 
-    changed, entity = naildown_entity(Setting, entity_dict, entity, 'present', module, check_type=True)
+    changed, entity = naildown_entity(Setting, entity_dict, entity, state, module, check_type=True)
```

Comment:
Now that we have state, we should add it to the `DOCUMENTATION`

Reply:
details, next you'll ask for an "absent" test :P

Reply:
I think, it is not possible to delete settings. So 'hidden present'(tm) is the only sensible option here.

Reply:
hah, this sounds about right. will fix that.

Reply:
Hrmpf. If i had let you run into writing the 'absent' test, you might have found a hidden feature of the foreman. Now, we settle with the boring stuff. ;)